NetworkOfPrivateClinics/NetworkOfPrivateClinics/AppointmentsValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NetworkOfPrivateClinics.CustomExceptions;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetworkOfPrivateClinics
+{
+    public class AppointmentsValidator
+    {
+        private TimeOnly startWorkingDay;
+        private TimeOnly endWorkingDay;
+        
+        public AppointmentsValidator(string startWorkingDay, string endWorkingDay) 
+        {
+            this.startWorkingDay = startWorkingDay.ToTimeOnly();
+            this.endWorkingDay = endWorkingDay.ToTimeOnly();
```

Comment:
don't use `this` keyword, use underscore prefix, like _startWorkingDay. Moreover make these fields readonly